sdk/eventhub/event-hubs/test/misc.spec.ts

```diff
@@ -192,6 +185,9 @@ describe("Misc tests #RunnableInBrowser", function(): void {
       const partitionId = hubInfo.partitionIds[0];
       const offset = (await client.getPartitionInformation(partitionId)).lastEnqueuedOffset;
       debug(`Partition ${partitionId} has last message with offset ${offset}.`);
+      breceiver = BatchingReceiver.create((client as any)._context, partitionId, EventPosition.fromOffset(offset));
+      let data = await breceiver.receive(5, 10);
+      should.equal(data.length, 0, "Unexpected to receive message before client sends it");
```

Comment:
`breceiver` is not used in this test. This might have gotten added when resolving merge conflicts